LayoutTests/http/wpt/webauthn/public-key-credential-get-success-hid.https-expected.txt

```diff
@@ -14,6 +14,4 @@ PASS PublicKeyCredential's [[get]] uses maxCredentialCountInList for batching wi
 PASS PublicKeyCredential's [[get]] with CTAP2â†’U2F downgrade skips wasteful CTAP2 tap (multi-credential).
 PASS PublicKeyCredential's [[get]] with CTAP2â†’U2F downgrade skips wasteful CTAP2 tap (single credential).
 PASS PublicKeyCredential's [[get]] with empty allowCredentials does NOT skip CTAP2 (resident key safety).
-PASS PublicKeyCredential's [[get]] with prf extension (1 salt) in a mock hid authenticator.
-PASS PublicKeyCredential's [[get]] with prf extension (2 salts) in a mock hid authenticator.
```

Comment:
Are we comfortable removing these two tests because you have added new API tests instead?